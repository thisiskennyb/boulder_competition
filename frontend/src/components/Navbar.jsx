import { Link } from "react-router-dom"

export default function Navbar() {
    return (
        <nav className="bg-indigo-200 p-4">
            <div className="container mx-auto flex justify-between items-center">
                <img src="https://placekitten.com/100/100" className="rounded-full mx-5"/>
                <Link to='profile'>
                    <h1 className="text-4xl text-violet-700">My Profile</h1>
                </Link>
                <Link to='/'>
                    <h1 className="text-4xl text-violet-700">Placeholder1</h1>
                </Link>
                <Link to='/'>
                    <h1 className="text-4xl text-violet-700">Placeholder2</h1>
                </Link>
                <Link to='/'>
                    <h1 className="text-4xl text-violet-700">Placeholder3</h1>
                </Link>
                <Link to='/'>
                    <h1 className="text-4xl text-violet-700">Placehodler4</h1>
                </Link>
                <Link to='logout'>
                    <h1 className="text-4xl text-violet-700 -mx-5">Logout</h1>
                </Link>

            </div>
          

        </nav>
    )
}