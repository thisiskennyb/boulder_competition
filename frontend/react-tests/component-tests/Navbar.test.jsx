import {expect, test } from "vitest"
import Navbar from "../../src/components/Navbar"

test("Navbar renders without crashing", () => {
    const component = <Navbar/>;
    expect(component).toMatchSnapshot();
})