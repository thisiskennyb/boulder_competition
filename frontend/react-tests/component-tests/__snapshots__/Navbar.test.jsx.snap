// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar renders without crashing 1`] = `<Navbar />`;
